core/modules/cms/store/page/actions.ts

```diff
@@ -41,8 +41,6 @@ const actions: ActionTree<CmsPageState, RootState> = {
       }
       return currentItems
     }
-
-    return dispatch('loadFromCache', { key, value, setCurrent })
```

Comment:
doesn't it break anything?

Reply:
For me, this line is unreachable. Maybe I don't notice here something.